cyoa.js

```diff
@@ -0,0 +1,180 @@
+function GetName(){
```

Comment:
Capitalized function names are usually reserved for constructor functions - we'll learn more about this next next week!